### PR TITLE
#7083 document logstash.yml ENV interpolation

### LINKED ISSUE #7083: Document using Environment Variable Interpolation in Settings File
Document the fix for #6785  as discussed in https://github.com/elastic/logstash/issues/6785#issuecomment-300983042

### PR DESCRIPTION
**Pull request by original-brownbear**
closes #7083 

Simple documentation of an example of the setting value interpolation.

### CODE REVIEW

File: docs/static/settings-file.asciidoc [hunk 1]
```diff
@@ -27,6 +27,22 @@ pipeline.batch.size: 125
...
+setting values.
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+pipeline:
+  batch:
+    size: ${BATCH_SIZE}
+    delay: ${BATCH_DELAY:5}
+node:
+  name: "node_${LS_NODE_NAME}"
```
Comment (jsvd) [inline docs/static/settings-file.asciidoc]: I'd also use the default value notation in the string interpolation example so it leaves no doubt that the default feature is supported in both scenarios.

**Comment by original-brownbear** [comment]
@jsvd added string interpolation default example :)

**Comment by jsvd** [comment]
squash and merge :shipit:

**Comment by original-brownbear** [comment]
@jsvd thanks  for the review!

**Pull request closed without merging**